Library/Homebrew/utils/versions.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Versions
```

Comment:
I think most of the stuff in this file could just live directly in `dev-cmd/bump.rb`. We also already have a `version.rb` and I worry that this could get confusing.

Reply:
Agreed.